### DISCUSSION TITLE
Calendar on Onyx Max Lumi 2

### DISCUSSION THREAD

**Discussion by gregor1991** [af/hyperpaper-planner#8] [General]
Hi,
I found your project via reddit and while I'm not an user of remarkable, but I find it very nice and would like to use it on my Max Lumi 2 if you are planning to publish a version for 2022. For me, the overall layout and graphic style very nice - the "cleanness" is very appealing. Also that you didn't implement - for me - unnecessary pages I really like :). For me personally, I also like the small size of the agenda. I don't have so much meetings, so for me it is more easy to work with the Notes & Journal section, while I have enough space on the left to include information about essential meetings.
While I like the overall style, I would also prefer lines and it would be great to have also a quaterly and weekly planner (but it is not 100% necessary). 
What would be very important for a complete year, would be way more note pages and project pages. Also, to make it more easy to reference to the notes and projects, a numbering would be great (while I really appreciate, that you have so much space for an individual title - great!).
Great work alltogether :).

P.S.: I'm new on github, so if it is better, to write these points as issues or pull request, please let me know. And if I can help anywhere - while I'm not really have to much knowledge about programming, please let me know (e.g. if you are planning to translate it into german or so).

**Comment by af** [comment]
Hi, thank you for this feedback! A few points and follow up questions:

* A few people have asked about a lined version, I'm planning to create one for 2022 (though my personal preference is still dots :)
* Any examples of how you would want the quarterly/weekly planners to look? I was considering both of these and I don't think they'd be too complex, but there are several possible approaches to them
* Extra note/project pages are no problem, it's definitely planned 👍 There actually are numbers on the notes pages and indexes but they're quite small, and I should make them more prominent

**Comment by af** [comment]
By the way, any feedback from the initial released version on the Max Lumi 2? Anything that could be improved for that device specifically? (I've never used one)

**Reply by gregor1991** [reply]
I did not find any issues/problems with the Max Lumi 2 - there is an error message, when I'm clicking on month, that don't exist, but I'm pretty sure, that's fine (since they don't exist). Other links etc. are working totally fine. Also the size of the dots/lines is appropriate in my opionion. I will try to take some pictures on Monday, I was on holiday this weekend and not 100% fit yet^^. I will also add some examples tomorrow :).

**Reply by af** [reply]
Thanks! The month error makes sense as long as you don't get it for Nov/Dec– those are the only months in the current pdf that should have functioning links.

PS new version coming soon, with lined pages :)

**Reply by gregor1991** [reply]
Nov/Dec are working perfectly fine! I'm really looking forward to it :).

**Reply by af** [reply]
Just published, give it a spin! https://github.com/af/planner-pdf/releases/tag/v0.6.0

**Reply by gregor1991** [reply]
Great, thank you! I downloaded it and tested it during the day. I really like it. 
The lines are great, the distance is totally fine on my 13" device. 
The agenda could be a bit smaller for me (similar to the old one), but I know, most want it in the big style as it is now. 
What is a bit strange: the Line is directly on the line of the timepoints - so I'm not sure, if the text, written on this line, belongs to the time before or after it. Hopefully, you know, what I mean. 
And I don't need this greyish background there, because I don't have the standard 9-5-working time. It would be great, if you can add small check boxes to the ToDo-Lines.
The additional note pages are working great. Maybe it would be helpfull to add links to the various note pages on the start page? 
I'm note entirely sure, if it would be helpfull to have link to notes on the daily pages. So you might spare the jump to the start page.
Regarding the quaterly and weekly planner:  For me, this would be a great weekly overview:
![image](https://user-images.githubusercontent.com/95493746/144899365-1b15d710-18de-4632-92b6-d4c1541e6587.png) (based on this post: https://tex.stackexchange.com/questions/332735/latex-calendar-layout-in-moleskine-leuchtturm-weekly-format)

And here, an idea for the quaterly overview (I did once a calendar manually - with a combination of LaTeX and Excel - not a great idea :D). It was heavily inspired by https://www.mydeepguide.com/product-page/mydailyorganizer (which has to much non necessary stuff inside, why I did not use this one and tried to do my own, but it did not work so great at all).
![image](https://user-images.githubusercontent.com/95493746/144900866-d8f6ed4a-fdce-43d7-8d40-8720bace4664.png)

**Reply by af** [reply]
More great feedback, thanks again!

> The agenda could be a bit smaller for me (similar to the old one), but I know, most want it in the big style as it is now

Yeah that seems to be the consensus. I've gotten used to the bigger height now and think I prefer it also.

> What is a bit strange: the Line is directly on the line of the timepoints - so I'm not sure, if the text, written on this line, belongs to the time before or after it. Hopefully, you know, what I mean.

I think I understand– the lines are meant to correspond to the time exactly on the hour. So for an event between 9am-10am you might draw a box between 9 and 10 lines and write inside it. It's similar to how calendar software like Google Calendar works in the weekly view, if that makes sense.

> And I don't need this greyish background there, because I don't have the standard 9-5-working time. 

Yeah this was an experiment, I've already improved it to be more flexible, but it'll be optional in the final version. Remind me if it's in the next version and I can export a version without it for you

> It would be great, if you can add small check boxes to the ToDo-Lines.

Should be no problem

> Maybe it would be helpfull to add links to the various note pages on the start page?

I was a wondering how to do this without the start page becoming too "busy", but it'd be useful. Will think about it some more

> I'm note entirely sure, if it would be helpfull to have link to notes on the daily pages.

Agreed, I was thinking the same. Will try to do this for the next version

> weekly/quarterly images

Thanks, very helpful to see visual examples. I'll see if I can incorporate something like them
